collect_app/src/test/java/org/odk/collect/android/configure/SettingsImporterTest.java

```diff
@@ -48,7 +48,7 @@ public void setup() {
         settingsValidator = mock(SettingsValidator.class);
         when(settingsValidator.isValid(any())).thenReturn(true);
 
-        importer = new SettingsImporter(generalSettings, adminSettings, (Settings generalSettings, Settings adminSettings) -> {}, settingsValidator, generalDefaults, adminDefaults, (key, newValue) -> {});
+        importer = new SettingsImporter(settingsProvider, (Settings generalSettings, Settings adminSettings) -> {}, settingsValidator, generalDefaults, adminDefaults, (key, newValue) -> {});
```

Comment:
Ah this is a nice change as we'll need the project ID for setting up jobs on import.